CHANGELOG.md

```diff
@@ -1185,6 +1185,7 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Report approved emails sent to BEIS users include a reminder that the report is going to be uploaded
 - Send a notification email to the report approver if the report upload fails
 - The latest generated spending breakdown CSVs can be downloaded from the Exports page
+- Spending breakdown email notification has a link to the Exports page instead of a direct download link
```

Comment:
Do you think we could make the commit message for this commit a bit more descriptive and include some of the context from this changelog entry? Looking back in a few months at the git log, I'm sure we'd appreciate it!

Reply:
Before I press the big button, would you have another look at the commit trail?